bee-storage/bee-storage-sled/Cargo.toml

```diff
@@ -0,0 +1,32 @@
+[package]
+name = "bee-storage-sled"
+version = "0.1.0"
+authors = ["IOTA Stiftung"]
+edition = "2018"
+description = "A bee-storage implementation for the Sled backend"
+readme = "README.md"
+repository = "https://github.com/iotaledger/bee"
+license = "Apache-2.0"
+keywords = ["iota", "tangle", "bee", "framework", "storage", "sled"]
```

Comment:
Maximum 5 keywords to publish a crate on crates.io. Yes I know rocksdb has 6, I just noticed it, and it's not published yet. Maybe remove `storage`. Also do the same for the rocksdb crate ?

Reply:
Not solved ?

Reply:
eventual consistency